src/Mapper.cpp

```diff
@@ -1346,3 +1346,223 @@ map<int, CGRANode*>* Mapper::getReorderPath(map<CGRANode*, int>* t_path) {
   return reorderPath;
 }
 
+
+// save the mapping results to json file for subsequent incremental mapping
+void Mapper::generateJSON4IncrementalMap(CGRA* t_cgra, DFG* t_dfg){
+        ofstream jsonFile("increMapInput.json", ios::out);
+        jsonFile<<"{"<<endl;
+        jsonFile<<"     \"Opt2TileXY\":{"<<endl;
+        int idx = 0;
+        for (DFGNode* dfgNode: t_dfg->nodes) {
+                // write dfgnodeID, mapped CGRANode X and Y coordinates
```

Comment:
`write` -> `Writes` and period at the end. Thanks! Same to the other comments.

Reply:
Has been corrected